### DISCUSSION TITLE
Using GLANCE7 in QGIS

### DISCUSSION THREAD

**Discussion by jo-spek** [davidfrantz/force#250] [Q&A]
Hi, since FORCE by default uses the GLANCE7 projection for products, I wanted to ask if anybody knows a simple implementation to use it in QGIS. I have been trying to add it as a custom coordinate reference system with the WKT strings from the GLANCE GitHub page, but it doesn't seem to work properly. Sometimes I get it to work with other rasters or polygons by randomly changing their projections back and forth, but I'd like to know what's actually happening. Unfortunately googling didn't help much, because GLANCE7 seems to be somewhat "off the grid" ;)

Anybody have advice? (I corrupted my QGIS installation at some point, but I was sure I had it properly repaired. So if GLANCE7 works like a charm in QGIS for you people, please let me know, too.)

**Comment by davidfrantz** [comment]
Hey @jo-spek,

I didn't observe any major problems so far. It simply works for me...

What happens on your end?

Cheers,
David
